hare3/hare_test.go

```diff
@@ -275,6 +300,21 @@ func (cl *lockstepCluster) addNode(n *node) {
 	cl.nodes = append(cl.nodes, n)
 }
 
+func (cl *lockstepCluster) partitionSigners() {
+	for i, signer := range cl.signers {
+		cl.nodes[i%len(cl.nodes)].register(signer.signer)
+	}
+}
+
+func (cl *lockstepCluster) addSigner(n int) *lockstepCluster {
+	last := len(cl.signers)
+	for i := last; i < last+n; i++ {
+		cl.signers = append(cl.signers, (&node{t: cl.t, i: i}).
+			withSigner().withAtx(cl.units.min, cl.units.max))
```

Comment:
It's hard to track the brackets, how about:
```suggestion
		n := (&node{t: cl.t, i: i}).withSigner().withAtx(cl.units.min, cl.units.max)
		cl.signers = append(cl.signers, n)
```